### PR TITLE
doc: add express middleware to README

### PR DESCRIPTION
**Pull request by ofrobots**
Documents #63.

### CODE REVIEW

**Comment by ofrobots** [comment]
@JustinBeckwith I am curious why the lint tests were failing in the samples on ce24c43. Presumably we updated the eslint got updated in a prior commit – but how did that commit land if the tests were failing? Either way, fixed things as part of this commit.

File: README.md [hunk 1]
```diff
@@ -96,6 +96,59 @@ logger.error('warp nacelles offline');
 logger.info('shields at 99%');
 ```
 
+### Using as an express middleware
+
+***NOTE: this feature is experimental. The API may change in a backwards
+incompatible way until this is deemed stable. Please do provide us feedback so
```
Comment (kjin) [inline README.md]: Nit: Remove `do`.

File: README.md [hunk 2]
```diff
@@ -96,6 +96,59 @@ logger.error('warp nacelles offline');
...
+that we can better refine this express integration.***
+
+We provide a middleware that can be used in an express application. Apart from
+being easy to use, this enables some more powerful features of Stackdriver
+Logging: request bundling.
+
+The middleware adds a `bunyan`-style log function to the `request` object. You
+can use this wherever you have access to the `request` object (`req` in the
+sample below). All log enties that are made on behalf of a specific request are
```
Comment (kjin) [inline README.md]: `entities`

File: README.md [hunk 3]
```diff
@@ -96,6 +96,59 @@ logger.error('warp nacelles offline');
...
+// Import express module and create an http server.
+const express = require('express');
+
+async function startServer() {
+  const {logger, mw} = await lb.express.middleware();
+  const app = express();
+
+  // Install the logging middleware. This ensures that a Bunyan-style `log`
+  // function is available on the `request` object. This should be the very
+  // first middleware you attach to your app.
```
Comment (kjin) [inline README.md]: Is this a strict requirement? I would expect that it's just that it needs to be installed before your own handlers.

File: README.md (hunk 1 above)
Comment (ofrobots) [reply README.md]: Un-do'd

File: README.md (hunk 2 above)
Comment (ofrobots) [reply README.md]: entries.

**Pull request merged**